var/spack/packages/apex/package.py

```diff
@@ -0,0 +1,30 @@
+from spack import *
+from spack.util.environment import *
+
+class Apex(Package):
+    homepage = "http://github.com/khuck/xpress-apex"
+    #url      = "http://github.com/khuck/xpress-apex/archive/v0.1-release-candidate.tar.gz"
+    url      = "http://github.com/khuck/xpress-apex"
+
+    #version('0.1', '6e039c224387348296739f6bf360d081')
+    version('master', branch='master', git='https://github.com/khuck/xpress-apex.git')
```

Comment:
Any chance you could make this more reproducible?  I generally point these at the date of the commit and just refer directly to the commit if there are no tags.  e.g.:
```
    version(''2015-10-21', git='https://github.com/khuck/xpress-apex.git', commit='abc123def456')
```
That would at least allow a user to add another custom version and have it installed simultaneously.

I'd like it if Spack just figured this out when people use branch heads for fetching (I could just query the repo for the commit and use that as the version), but I haven't done that yet.